topo_processor/stac/tests/item_test.py

```diff
@@ -32,5 +30,6 @@ def test_duplicate_asset():
             needs_upload=True,
         ),
     )
-    with pytest.raises(Exception, match=r"./collection_title/item_id.tiff already exists."):
+    item.assets["source"].needs_upload = False
+    with pytest.raises(Exception, match=r"."):
```

Comment:
matching any exception is not a good idea for tests

Reply:
changed back to with pytest.raises(Exception, match=r"./collection_title/item_id.tiff already exists."):